### PR TITLE
Add institutional events tab accesses

### PR DESCRIPTION
**Pull request by kevin-verschaeve**

### CODE REVIEW

File: src/Controller/EnMarche/InstitutionalEvents/DelegatedReferentInstitutionalEventsController.php [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+use App\Controller\CanaryControllerTrait;
+use App\Entity\MyTeam\DelegatedAccess;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Security;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\Routing\Annotation\Route;
+
+/**
+ * @Route("/espace-referent-partage/{delegated_access_uuid}/evenements-institutionnels", name="app_referent_institutional_events_delegated_", methods={"GET"})
+ *
+ * @Security("is_granted('ROLE_DELEGATED_REFERENT')")
```
Comment (ottaviano) [inline src/Controller/EnMarche/InstitutionalEvents/DelegatedReferentInstitutionalEventsController.php:15]: do we need to check `EVENT` access here?

File: src/Controller/EnMarche/ReferentController.php [hunk 2]
```diff
@@ -2,19 +2,12 @@
...
-use App\Address\GeoCoder;
 use App\Entity\Adherent;
-use App\Entity\InstitutionalEvent;
 use App\Entity\ReferentOrganizationalChart\PersonOrganizationalChartItem;
-use App\Form\InstitutionalEventCommandType;
 use App\Form\ReferentPersonLinkType;
-use App\InstitutionalEvent\InstitutionalEventCommand;
-use App\InstitutionalEvent\InstitutionalEventCommandHandler;
 use App\Intl\FranceCitiesBundle;
-use App\Referent\ManagedInstitutionalEventsExporter;
```
Comment (ottaviano) [inline src/Controller/EnMarche/ReferentController.php:14]: could we remove the file `ManagedInstitutionalEventsExporter.php`?

File: src/Controller/EnMarche/InstitutionalEvents/DelegatedReferentInstitutionalEventsController.php (hunk 1 above)
Comment (kevin-verschaeve) [reply src/Controller/EnMarche/InstitutionalEvents/DelegatedReferentInstitutionalEventsController.php]: yes, fixed

File: src/Controller/EnMarche/ReferentController.php (hunk 2 above)
Comment (kevin-verschaeve) [reply src/Controller/EnMarche/ReferentController.php:14]: indeed, it seems not used anymore, removed

**Pull request merged**